Assets/Scripts/MainMenu.cs

```diff
@@ -0,0 +1,99 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MainMenu : MonoBehaviour
+{
+    public GameObject pivotPoint; //For camera
+    public float rotationSpeed;
+    public float creditsSpeed;
+
+    bool isCreditsWindowOpen;
+
+    public Scene mainMenuScene;
+    public GameObject confirmationWindow;
+    public GameObject creditsWindow;
+    public GameObject backCreditsButton;
+    public GameObject settingsWindow;
+    public GameObject buttonsInMenu;
```

Comment:
Is all `public` necessary? If not - change it to private. You can use` [SerializeField]` above so your variable will be seen in the inspector.
Example:
```
 [SerializeField]
 private GameObject buttonsInMenu
```